client/src/main/java/io/pravega/client/stream/impl/ReaderGroupImpl.java

```diff
@@ -117,6 +124,21 @@ public void readerOffline(String readerId, Position lastPosition) {
         return synchronizer.getState().getStreamNames();
     }
 
+    public ReaderGroupState updateState(ReaderGroupState.ConfigState configState, ReaderGroupState state, AtomicBoolean b) {
```

Comment:
Could we call this `updateConfigState` since this name can be easily misunderstood with "updateState" in statesynchronizer

Reply:
Yes this change can be made.